tutor/src/models/course/information.js

```diff
@@ -68,14 +69,23 @@ export default {
     title:      'Anatomy & Physiology',
     subject:    SUBJECTS.ANATOMY_PHYSIOLOGY,
   },
+};
+
+each(BOOKS, (properties, code) => properties.code = code);
```

Comment:
This syntax that is the exact opposite order of ruby's always confused me

Reply:
yeah I know, same here.   This is where the `code` is set that you thought might be missing in the other comment.  Rather than have it typed out on each, I did this so it'll always match the key.